docs/unlocked_package.rst

```diff
@@ -3,105 +3,139 @@ Release an Unlocked Package
 
 While CumulusCI was originally created to develop managed packages, it can also be used to develop and release `unlocked packages <https://developer.salesforce.com/docs/atlas.en-us.sfdx_dev.meta/sfdx_dev/sfdx_dev_unlocked_pkg_intro.htm>`_.
```

Comment:
Are we capitalizing Unlocked Packages everywhere or not?

Reply:
The officials docs are now using lowercase; I'll harmonize.